bfd/ChangeLog

```diff
@@ -1,3 +1,7 @@
+2022-01-29  Charlie Keaney  <charlie.keaney@embecosm.com>
+
+	* bfd/elfxx-riscv.c: Fixed typo in error message.
+
```

Comment:
I would probably keep this in a `ChangeLog.COREV` to keep in sync with how e.g. `gas/testsuite/ChangeLog.COREV` would exist.